react/src/lib/ListItem/index.js

```diff
@@ -200,8 +200,8 @@ class ListItem extends React.Component {
               contextProps.id = this.props.id || id;
               contextProps.uniqueKey = navKey || contextProps.id;
               contextProps.type = type || (listContext && listContext.type);
-              contextProps.focus = focus || (listContext && listContext.focus === contextProps.uniqueKey);
-              contextProps.active = active || (listContext && listContext.active === contextProps.uniqueKey);
+              contextProps.focus = focus || (listContext && makeEventKey(listContext.focus) === contextProps.uniqueKey);
```

Comment:
This code is problematic because makeEventKey forces kebab case when the id may not be kebab.  It's meant to be ran only if the eventKey is provided.  I'm sure there's a better way to handle this, but currently this breaks the code when relying on the id from react-uid.